site/static/shacl-extensions

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Eclipse RDF4J SHACL Extensions</title>
+</head>
+<body>
+<ul>
+<li>
+<a href="shacl-extensions.html">Specification</a>
+</li>
+<li>
+<a href="shacl-extensions.ttl">Namespace (turtle)</a>
+</li>
+</ul>
+</html>
```

Comment:
I haven't tested if this work, or if it just ends up being downloaded.

Reply:
I don't understand the setup. Why have an html file that links to another html file and a turtle file? Why not simply have a link to the turtle file _inside_ the specification? For that matter, why have a turtle file at all? 

Given that we have multiple files relating to the same thing, I'd prefer if we could organize this a little neater: a subdirectory, for one thing.

Reply:
Goal was that if you go to the namespace url, you end up at a page where you can choose if you want to read the spec or download the turtle file. The turtle file contains the schema for the namespace. We could also add the SHACL shapes for validating our extension to that turtle file. That's what SHACL does with their schema file.

Reply:
Fair enough. I've reorganized this slightly - the html file with the links has been moved to /content and changed to markdown (bonus is that it now uses our website theme). The spec and the schema file have been added to a subdir to keep things organized (also because in future we may want to publish more specs this way). Tested locally, the namespace resolves to the link page, so all works as intended.